docs/examples/101/basic-batch-account/main.bicep

```diff
@@ -0,0 +1,20 @@
+param BatchaccountName string
+param allocationMode string {
+  default: 'BatchService'
+  allowed: [
+    'BatchService'
+    'Usersubscription'
+  ]
+}
+
+param location string = resourceGroup().location
+
+resource batchaccount 'Microsoft.Batch/batchAccounts@2020-09-01' = {

```

Comment:
change the resource identifier to `batchAccount`

Reply:
@alex-frankel , Updated resource identifier as per review.